packages/server/src/main.ts

```diff
@@ -8,7 +8,7 @@ const prisma = new PrismaClient();
 async function bootstrap(): Promise<void> {
   const app = await NestFactory.create<NestExpressApplication>(AppModule);
   app.useGlobalPipes(new ValidationPipe());
-  await app.listen(3001);
+  await app.listen(3000);
```

Comment:
Deployment is using 3001

```suggestion
  await app.listen(3001);
```